src/Cli/dotnet/commands/dotnet-nuget/NuGetCommandParser.cs

```diff
@@ -1,61 +1,104 @@
-                    Create.Option("-h|--help", Parser.CompletionOnlyDescription),
-                    Create.Option("--force-english-output", Parser.CompletionOnlyDescription),
-                    Create.Option("-s|--source", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("--non-interactive", Parser.CompletionOnlyDescription),
-                    Create.Option("-k|--api-key", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("--no-service-endpoint", Parser.CompletionOnlyDescription),
-                    Create.Option("--interactive", Parser.CompletionOnlyDescription)),
-                Create.Command(
-                    "locals",
-                    Parser.CompletionOnlyDescription,
-                    Accept.AnyOneOf(
-                        "all",
-                        "http-cache",
-                        "global-packages",
-                        "plugins-cache",
-                        "temp"),
-                    Create.Option("-h|--help", Parser.CompletionOnlyDescription),
-                    Create.Option("--force-english-output", Parser.CompletionOnlyDescription),
-                    Create.Option("-c|--clear", Parser.CompletionOnlyDescription),
-                    Create.Option("-l|--list", Parser.CompletionOnlyDescription)),
-                Create.Command(
-                    "push",
-                    Parser.CompletionOnlyDescription,
-                    Accept.OneOrMoreArguments(),
-                    Create.Option("-h|--help", Parser.CompletionOnlyDescription),
-                    Create.Option("--force-english-output", Parser.CompletionOnlyDescription),
-                    Create.Option("-s|--source", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("-ss|--symbol-source", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("-t|--timeout", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("-k|--api-key", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("-sk|--symbol-api-key", Parser.CompletionOnlyDescription, Accept.ExactlyOneArgument()),
-                    Create.Option("-d|--disable-buffering", Parser.CompletionOnlyDescription),
-                    Create.Option("-n|--no-symbols", Parser.CompletionOnlyDescription),
-                    Create.Option("--no-service-endpoint", Parser.CompletionOnlyDescription),
-                    Create.Option("--interactive", Parser.CompletionOnlyDescription),
-                    Create.Option("--skip-duplicate", Parser.CompletionOnlyDescription)
-                    ));
+        private static readonly string CompletionOnlyDescription = "-";
+
+        public static Command GetCommand()
+        {
+            var command = new Command("nuget", CompletionOnlyDescription);
+
```

Comment:
I don't see `command.AddOption(new Option(new string[] { "-h", "--help" }, CompletionOnlyDescription);` option for all the `dotnet nuget` commands. Am I missing anything?

Reply:
System.CommandLine has help built in, so it's specified in the parser: https://github.com/dotnet/sdk/pull/14379/files#diff-75f24d684a51fe02f83d64c70e9cd87276f55132b61facd9fd41bb0b5d1da847R134